src/forces.f90

```diff
@@ -13,522 +13,832 @@
+             i=icvrt(iv)-ystart(1)+1
+             do k=1,ysize(3)
+                kk=ystart(3)-1+k
+                fcvx=zero
+                fcvy=zero
+                fprx=zero
+                fdix=zero
+                fdiy=zero
+                do j=jcvlw_ly(iv),jcvup_ly(iv)-1
+                   !momentum flux
+                   uxmid = half*(ux2(i,j,k)+ux2(i,j+1,k))
+                   uymid = half*(uy2(i,j,k)+uy2(i,j+1,k))
+                   fcvx= fcvx +uxmid*uxmid*del_y(j)
+                   fcvy= fcvy +uxmid*uymid*del_y(j)
+ 
+                   !pressure
+                   prmid=half*(ppi2(i,j,k)+ppi2(i,j+1,k))
+                   fprx = fprx -prmid*del_y(j)
+ 
+                   !viscous term
+                   dudxmid = half*(ta2(i,j,k)+ta2(i,j+1,k))
+                   dudymid = half*(td2(i,j,k)+td2(i,j+1,k))
+                   dvdxmid = half*(tb2(i,j,k)+tb2(i,j+1,k))
+                   fdix = fdix +two*xnu*dudxmid*del_y(j)
+                   fdiy = fdiy +xnu*(dvdxmid+dudymid)*del_y(j)
+                enddo
+                tconvxl(kk)=tconvxl(kk)+fcvx
+                tconvyl(kk)=tconvyl(kk)+fcvy
+                tpresxl(kk)=tpresxl(kk)+fprx
+                tdiffxl(kk)=tdiffxl(kk)+fdix
+                tdiffyl(kk)=tdiffyl(kk)+fdiy
+             enddo
+          endif
+          call MPI_ALLREDUCE(tconvxl,tconvx,nz,real_type,MPI_SUM,MPI_COMM_WORLD,code)
+          call MPI_ALLREDUCE(tconvyl,tconvy,nz,real_type,MPI_SUM,MPI_COMM_WORLD,code)
+          call MPI_ALLREDUCE(tpresxl,tpresx,nz,real_type,MPI_SUM,MPI_COMM_WORLD,code)
+          call MPI_ALLREDUCE(tpresyl,tpresy,nz,real_type,MPI_SUM,MPI_COMM_WORLD,code)
+          call MPI_ALLREDUCE(tdiffxl,tdiffx,nz,real_type,MPI_SUM,MPI_COMM_WORLD,code)
+          call MPI_ALLREDUCE(tdiffyl,tdiffy,nz,real_type,MPI_SUM,MPI_COMM_WORLD,code)
+ 
+          do k=1,zsize(3)
+ 
+             tpresx(k)=tpresx(k)/dt
+             tpresy(k)=tpresy(k)/dt
+ 
+             xmom    = tunstx(k)+tconvx(k)
+             ymom    = tunsty(k)+tconvy(k)
+             xDrag(k) = two*(tdiffx(k)+tpresx(k)-xmom)
+             yLift(k) = two*(tdiffy(k)+tpresy(k)-ymom)
+ 
           enddo
+ 
+          !Edited by F. Schuch
+          xDrag_mean = sum(xDrag(:))/real(nz,mytype)
+          yLift_mean = sum(yLift(:))/real(nz,mytype)
+ 
+          !     if ((itime==ifirst).or.(itime==0)) then
+          !        if (nrank .eq. 0) then
+          !        write(filename,"('aerof',I1.1)") iv
+          !        open(38+(iv-1),file=filename,status='unknown',form='formatted')
+          !        endif
+          !     endif
+          if (nrank .eq. 0) then
+             write(38,*) t,xDrag_mean,yLift_mean
+             call flush(38)
+          endif
+          if (mod(itime, icheckpoint).eq.0) then
+             if (nrank .eq. 0) then
+                write(filename,"('forces.dat',I7.7)") itime
+                call system("cp forces.dat " //filename)
```

Comment:
Why would we call `system` instead of `execute_command_line` ?

Reply:
Again, the latest code also calls system so I used the same.